freqtrade/exchange/bittrex.py

```diff
@@ -122,9 +122,10 @@ def get_ticker(self, pair: str, refresh: Optional[bool] = True) -> dict:
                 raise OperationalException('{message} params=({pair})'.format(
                     message=data['message'],
                     pair=pair))
-
+            keys = ['Bid', 'Ask', 'Last']
             if not data.get('result') or\
-                    not all(key in data.get('result', {}) for key in ['Bid', 'Ask', 'Last']):
+                    not all(key in data.get('result', {}) for key in keys) or\
+                    not all(data.get('result', {})[key] is not None for key in keys):
```

Comment:
hmm @vertti can you figure out a efficient way to do the above? I have a gut feeling that looping the results multiple times per `get_ticker` call might not be optimal in the long run (If I understood the above behaviour correctly)

Reply:
This is only checking 3 fields in a dict, this is not the global ticker_history, only the latest ticker containing 'Bid','Ask' and 'Last'

Reply:
@jblestang ah, I stand corrected. Thanks 👍